assets/admin/app.js

```diff
@@ -0,0 +1,3 @@
+// Add additional extensions here:
+
+// Implement custom extensions here:
```

Comment:
maybe just the following?

```suggestion
// Add project specific javascript code here:
```